### PR TITLE
Setup GitHub Actions CI

### PR DESCRIPTION
**Pull request by utilForever**
This revision includes:
- Setup GitHub Actions CI
  - Add GitHub Actions configuration file for Rust build
  - Add unit test code for function 'hello()'

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  rustfmt:
+    name: 🗊 Formatting
```
Comment (plzfday) [inline .github/workflows/rust.yml:13]: Is it supposed to be an emoji?

Comment (utilForever) [reply .github/workflows/rust.yml:13]: Yes, it is.

**Comment by w8385** [comment]
image

**Review by w8385 — APPROVED** [review]

**Review by plzfday — APPROVED** [review]
Great!

**Pull request merged**